site/docs/design_tokens/box-shadow.mdx

```diff
@@ -0,0 +1,32 @@
+---
+name: Box shadow
+menu: Design tokens
+route: /design-tokens/box-shadow
+---
+
+import LargeParagraph from "../../src/components/LargeParagraph";
+
+# Box shadow
+
+<LargeParagraph>Shadows can be used under certain elements to create visual distinction for content that otherwise would be hard to distinguish for the visually impaired or remain unnoticed.
+</LargeParagraph>
+
+Box-shadows are used to cast shadows from interface elements to backgrounds or other elements to allow visual cohesion and a more accessible experience.
+
+## Principles
+- **Shadows are not used to signify elevation in HDS.** Elements that require distinction, can however have shadows applied.
+- **Do not use shadows to highlight elements.** Elements with shadow around them should only have it for visual clarity.
+- **Never use shadows on typography (heading nor body).** Primary typographic contrast should always rely on colour contrast. 
+- **Never use shadows for effect.** For the City of Helsinki brand to remain intact, do not rely on style based effects of any kind.
+
+## Usage
+
+Shadows should be used very sparingly if at all and always only to allow greater visual contrast between elements. **Never use shadows as effects**.
+
+## Shadow tokens
+
+CSS              | SASS              | RGBa                         | Example                                                                                                                                                                    |
+---------------- | ----------------- | ---------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+--shadow-s       | $shadow-s         | 0 0 10px rgba(0,0,0,.07);    | <span style="margin: 20px 0; box-shadow: 0px 0px 10px rgba(0, 0, 0, .07); background-color: var(--color-fog); display: block; height: 60px; width: 60px;"></span>          |
```

Comment:
Would it be possible to use the token in the example instead of hardcoded values?

Reply:
- [ ] Use tokens